config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
```

Comment:
Hmm, where we want to put this exercise just depends on: Do we think `grains` is well-placed in this track, given its difficulty relative to the exercises around it?

Maybe the picture would be easier to see if we have a single PR proposing difficulty numbers for all the exercises, without adding topics to most of them yet (it's what #345 suggests to do).

For example, is there any exercise before `grains` that we would rate a 3? Any exercise after `grains` that we would rate a 1? If so, that is a bit strange, and then maybe some of those exercises should get moved.